tests/test_getURL.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import ndio.remote.neurodata as neurodata
+import test_settings
+
+neurodata_token = "1524d74ec9a915536061af8fd5b51f2cde7de9a5"
```

Comment:
These should not be explicit, but rather read in from the file at /tmp/token_super or token_test. I would prefer to keep them actually in test_settings overall, similar to how HOSTNAME is.